cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -90,6 +99,21 @@ class TowerVariableValue(models.Model):
         ),
     ]
 
+    @api.depends("variable_id", "option_id")
```

Comment:
You should depend on the fields that are used to compute the value. Check the next comment->